### DISCUSSION TITLE
How can I switch account

### DISCUSSION THREAD

**Discussion by Sadlysius** [z-huang/music#1025] [Q&A]
I have multiple brand account, how can I switch to my other brand/channel account?

**Comment by z-huang** [comment]
Isn't your music stored in one account?

**Reply by Sadlysius** [reply]
If you have multiple brand account they are stored separately

**Reply by z-huang** [reply]
I mean, under normal circumstances, people save their songs in a single account even if they have multiple accounts. This makes management easier. Therefore, this app may not support switching accounts.

**Reply by Sadlysius** [reply]
I have multiple channels under one account [Screenshot_20230905-103641_InnerTune~2.png](https://github.com/z-huang/InnerTune/assets/108415154/9530b300-4e85-406d-a779-86ba42d9f0d4)

**Reply by z-huang** [reply]
You're recommended to create an issue for this. Things in discussion section are easy to be forgotten.
